[RHACS] ROX-20244: Adding Inviting users to RHACS documentation (#66457)
openshift/openshift-docs: configuration/inviting-users-to-rhacs.adoc

:_content-type: ASSEMBLY
[id="inviting-users-to-rhacs"]
= Inviting users to {product-title-short}
include::modules/common-attributes.adoc[]
:context: inviting-users

toc::[]

[role="_abstract"]

> Inviting users to {rh-rhacs-first} ensures that the right users have the appropriate access rights within your cluster. The following section provides an overview of how to invite one or more users by assigning roles and defining the authentication provider.

Reviewer: * Suggestion to put focus on user's action: "By inviting users to {rh-rhacs-first}, you can ensure that"
* Per ISG/minimalism, should avoid self-referential references, like "the following section". Suggest to rewrite this to avoid

```suggestion
By inviting users to {rh-rhacs-first}, you can ensure that the right users have the appropriate access rights within your cluster. You can invite one or more users by assigning roles and defining the authentication provider.
```
